riotfile.py

```diff
@@ -0,0 +1,41 @@
+from riot import Venv
+
+venv = Venv(
+    pys=["3.9"],
+    venvs=[
+        Venv(
+            pkgs={
+                "black": "==20.8b1",
+            },
+            venvs=[
+                Venv(
+                    name="check_fmt",
+                    command="black --check .",
+                ),
+                Venv(
+                    name="fmt",
+                    command="black .",
+                ),
```

Comment:
is this extra?

Reply:
this is for running locally so you can do `riot run -s fmt` and it'll format using black